s3-uploader/runtimes/dotnet7_aot/build.sh

```diff
@@ -0,0 +1,7 @@
+SOURCE=${BASH_SOURCE[0]}
+DIR_NAME=$(dirname ${SOURCE})
+rm ${DIR_NAME}/code.zip 2> /dev/null
+
+docker build ${DIR_NAME} -t maxday/dotnet6
```

Comment:
I think you meant `dotnet7` ?

Reply:
I did, and changed this too 🤔. Must have been an issue saving, I can resolve this.